docker-compose.yml

```diff
@@ -185,7 +186,6 @@ services:
   # used as an S3 service mock
   minio:
     image: minio/minio:RELEASE.2021-08-31T05-46-54Z
-    profiles: ["mock-services"]
```

Comment:
Could this change mean that we could be running mino where we don't mean to be (i.e. outside of a test context)? Do the other change work without changing this approach to only running mino in a test context?